metricbeat/module/munin/_meta/fields.yml

```diff
@@ -4,8 +4,16 @@
     Munin node metrics exporter
   release: beta
   fields:
+    - name: munin.metrics.*.*
+      type: object
+      object_type: double
+      object_type_mapping_type: '*'
+      description: >
+          munin.metrics contains metrics exposed by a munin node agent
+    - name: munin.metrics.*
```

Comment:
This is interesting. So the first entries always have to be objects? What happens if we leave out this config?

Reply:
Yes, first-level entries should always be objects. Without this config we wouldn't be enforcing that.